pkgs/top-level/all-packages.nix

```
@@ -5949,6 +5949,8 @@ in
 
   rc = callPackage ../shells/rc { };
 
+  rdbtools = callPackage ../development/tools/rdbtools { };
```

set python version here
```suggestion
  rdbtools = callPackage ../development/tools/rdbtools {
    python = python3;
  };
```

---

python2 is near EOL, use python3 please. I know im going to have to revisit this in a few weeks if i dont